semantic-core/src/Data/Core.hs

```diff
@@ -221,10 +222,12 @@ instance HFunctor Ann
 instance RightModule Ann where
   Ann l b >>=* f = Ann l (b >>= f)
 
-
```

Comment:
Please don’t delete my spacing! I separate things for readability’s sake.

Reply:
Oops!